front/src/utils/withIntlAsProp.js

```diff
@@ -0,0 +1,7 @@
+import { IntlContext } from 'preact-i18n';
+
+const withIntlAsProp = WrappedComponent => props => (
+  <IntlContext.Consumer>{({ intl }) => <WrappedComponent {...{ ...props, intl }} />}</IntlContext.Consumer>
```

Comment:
With `{...{ ...props, intl }}`, you copy twice the object.
I think `<WrappedComponent {...props} intl={intl} />` is more efficient, (or `<WrappedComponent {...props, intl} />` only once).